tests/test_task_api.py

```diff
@@ -36,7 +36,9 @@ async def test_task_get_list(db, client, task_factory, MockCurrentUser):
         assert data[2]["id"] == t2.id
 
 
-async def test_background_collection(db, dummy_task_package):
+async def test_background_collection(
+    db, registered_client, dummy_task_package
```

Comment:
why registered client?

Reply:
Because with `client` we get `assert 401 == 200` at line 67 (unauthorized).

Reply:
Use with MockCurrentUser